stubs/redis/redis/commands/core.pyi

```diff
@@ -159,7 +416,7 @@ class CoreCommands(Generic[_StrType]):
     @overload
     def bzpopmin(self, keys: _Key | Iterable[_Key], timeout: float) -> tuple[_StrType, _StrType, float] | None: ...
     @overload
-    def zrange(
+    def zrange(  # type: ignore[overload]
```

Comment:
Why is the ignore comment needed now, but wasn't needed before?

Reply:
I don't remember. Let's try without.